### PR TITLE
feat: Add Experitest as cloud provider

### PR DESCRIPTION
**Pull request by ArielExperitest**
Added Experitest to desktop Appium UI as Cloud provider.
Appium Desktop users will be able to integrate with the Experitest Cloud.
Experitest users will need only access key and there Cloud URL.

If Experitest integration is chosen the user will need to enter:
- 'URL' - Full Experitest Cloud URL (http://example.experitest.com:7777)
- accessKey - User credentials

screen-new-session-cloud-providers

### CODE REVIEW

File: app/renderer/components/Session/ServerTabExperitest.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+    const placeholderUrl = 'https://example.experitest.com';
+
+    return <Form>
+      <Row gutter={8}>
+        <Col span={24}>
+          <FormItem>
+            <Input className={SessionStyles.customServerInputLeft} id='ExperitestServerUrl' placeholder={placeholderUrl}
+              addonBefore={t('Experitest Url')} value={server.experitest.url}
+              onChange={(e) => setServerParam('url', e.target.value)} />
```
Comment (mykola-mokhnach) [inline app/renderer/components/Session/ServerTabExperitest.js]: I'd rather use `evt` to not mix it with `error`

File: app/renderer/components/Session/ServerTabExperitest.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+          </FormItem>
+        </Col>
+      </Row>
+      <Row gutter={8}>
+        <Col span={24}>
+          <FormItem>
+            <Input className={SessionStyles.customServerInputLeft} id='ExperitestServerAccessKey' placeholder={accessKeyPlaceholder}
+              addonBefore={t('Experitest AccessKey')} value={server.experitest.accessKey}
```
Comment (mykola-mokhnach) [inline app/renderer/components/Session/ServerTabExperitest.js]: please add `Experitest AccessKey` string to resources

File: app/renderer/components/Session/ServerTabExperitest.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    const {server, setServerParam, t} = this.props;
+
+    const placeholderUrl = 'https://example.experitest.com';
+
+    return <Form>
+      <Row gutter={8}>
+        <Col span={24}>
+          <FormItem>
+            <Input className={SessionStyles.customServerInputLeft} id='ExperitestServerUrl' placeholder={placeholderUrl}
+              addonBefore={t('Experitest Url')} value={server.experitest.url}
```
Comment (mykola-mokhnach) [inline app/renderer/components/Session/ServerTabExperitest.js]: Please add `Experitest Url` string to resources

Comment (ArielExperitest) [reply app/renderer/components/Session/ServerTabExperitest.js]: Done.

File: app/renderer/components/Session/ServerTabExperitest.js (hunk 2 above)
Comment (ArielExperitest) [reply app/renderer/components/Session/ServerTabExperitest.js]: Done.

File: app/renderer/components/Session/ServerTabExperitest.js (hunk 1 above)
Comment (ArielExperitest) [reply app/renderer/components/Session/ServerTabExperitest.js]: Done.

File: app/renderer/actions/Session.js [hunk 4]
```diff
@@ -321,6 +321,23 @@ export function newSession (caps, attachSessId = null) {
         }
         https = session.server.testingbot.ssl = true;
         break;
+      case ServerTypes.experitest: {
+        if (!session.server.experitest.url || !session.server.experitest.accessKey) {
+          notification.error({
+            message: 'Error',
```
Comment (mykola-mokhnach) [inline app/renderer/actions/Session.js]: should the `Error` label be translatable?

**Comment by mykola-mokhnach** [comment]
It would be nice to have the screenshot of the newly added dialog attached to the PR

File: app/renderer/actions/Session.js (hunk 4 above)
Comment (ArielExperitest) [reply app/renderer/actions/Session.js]: Thanks, you are right!

**Pull request merged**